storage/redis_controller.go

```diff
@@ -5,15 +5,13 @@ import (
 	"sync/atomic"
 	"time"
 
-	redis "github.com/go-redis/redis/v8"
-
 	"github.com/TykTechnologies/tyk/config"
 )
 
 type RedisController struct {
```

Comment:
Eventually I'd like to do two things:

- NewLoggedStorage(storage.Storage) storage.Storage - add logging to all storage calls by wrapping the storage,
- NewToggledStorage(storage.Storage) *toggledStorage (storage.Storage + enable/disable)

This would decouple current logging and enable/disable toggles that are bolted onto the storage currently, moving towards single-responsibility objects.

Out of scope for now as it would be a breaking change.